Casks/v/visit.rb

```diff
@@ -1,16 +1,16 @@
 cask "visit" do
-  on_high_sierra :or_older do
-    version "3.1.1"
-    sha256 "4213daed23de17ee8bcfba779a96cce3ef92d3075ae666f7aeaffa824d484924"
+  on_big_sur :or_older do
+    version "3.3.2"
+    sha256 "f406850b21ddc16a6ca636b5c9087b41baeb5460ac92a9b140194c010eb285b3"
 
-    url "https://github.com/visit-dav/visit/releases/download/v#{version}/visit#{version}.darwin-x86_64-10.13.dmg",
+    url "https://github.com/visit-dav/visit/releases/download/v3.3.2/Visit-3.3.2.dmg",
```

Comment:
```suggestion
    url "https://github.com/visit-dav/visit/releases/download/v#{version}/Visit-#{version}.dmg",
```

Unless there's a particular reason to hardcode the version